recipe/meta.yaml

```diff
@@ -23,15 +23,15 @@ requirements:
   run:
     - anndata >=0.9.1
     - click
-    - dask-core >=2024.4.1
+    - dask-core >=2024.4.1,<2024.11.0
```

Comment:
Looks good to me. The latest version of `spatialdata` works up to `2024.12.1`, so this upper bound could be used. But since `spatialdata==0.2.4` was released in November, it's also an option to use the upper bound you specified for `0.2.3` and then relax it with `0.2.4`.

Reply:
> The latest version of `spatialdata` works up to `2024.12.1`, so this upper bound could be used. 

Thanks for the extra context

> But since `spatialdata==0.2.4` was released in November, it's also an option to use the upper bound you specified for `0.2.3` and then relax it with `0.2.4`.

Right, my plan is to go step-by-step. First get 0.2.3 functional again. Then release 0.2.5. And then finally 0.3.0 (which as you pointed out in that PR is blocked by xarray at the moment anyways). I'll update the lower and upper bounds as I go

Reply:
Thanks a lot!